modules/contact-form/admin.php

```diff
@@ -784,6 +895,44 @@ function grunion_ajax_spam() {
 		$status_html .= '</span></a></li>';
 	}
 
+	if ( isset( $status['responded'] ) ) {
+		$status_html .= '<li><a href="edit.php?post_status=responded&amp;post_type=feedback"';
+		if ( $current_menu == 'responded' ) {
```

Comment:
I would use Yoda conditions here as well.

```suggestion
		if ( 'responded' === $current_menu ) {
```

That said, and like above, I wonder if it may be worth refactoring this a bit so we don't have to keep repeating ourselves below.